src/Pages/login.jsx

```diff
@@ -0,0 +1,111 @@
+import { event } from "jquery";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default () => {
+  const navigate = useNavigate();
+  const [userName, setUserName] = useState(null);
+  const [password, setPassword] = useState(null);
+
+  const [isUsernameActivated, setUsernameActivated] = useState(false);
+  const [isPasswordActivated, setPasswordActivated] = useState(false);
+
+  const [unalert, setUnAlert] = useState(null);
+  const [pwalert, setPwAlert] = useState(null);
+
+  function handleUsername(event){
+    setUserName(event.target.value);
+    
+    checkUsername(userName);
+  
+  }
+
+  function handleUnBlur() {
+    setUsernameActivated(true)
+    checkUsername(userName)
+  }
+    function handlePwBlur() {
+      setPasswordActivated(true);
+      checkPassword(password)
+    }
+
+  function checkUsername(userName) {
```

Comment:
This will not be possible with arrow functions. But as a rule of thumb we should always keep in mind to define a function before it's used.

Reply:
Fixed e97eae75b859e2f1f70c09d6abe315d63c384f20